index.html

```diff
@@ -6,7 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="index.css">
   <title>Awesome-books-coker</title>
-  <script type="module" src="index.js"></script>
+  <script type="module" defer src="index.js"></script>
 </head>
 <body>
   <header>
```

Comment:
Everything is working well in your project , however when the page opens , I can see this page below . Which is not a good way of doing thing , Kindly make sure that when the page reload the user can only see on section not all at a time 😄 
![Capture](https://github.com/coker2019/Awesome-books-Es6/assets/80458199/d34c598d-6c5f-4983-bc05-44823cdbed79)